src/consensus/softforks.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2015 Eric Lombrozo
+// Copyright (c) 2009-2015 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_SOFTFORKS_H
+#define BITCOIN_SOFTFORKS_H
+
+class CBlockIndex;
+namespace Consensus {
+
+struct Params;
+namespace SoftForks {
+
+enum Rule
+{
+    BIP16,
+    BIP30,
+    BIP34,
+    BIP42,
+    BIP62,
+    BIP65,
+    BIP66,
+    BIP68,
```

Comment:
Again, there's not need to include future rule changes (their own PRs can do it if they get merged after this, maybe with the exception of BIP65 for which most of the code has been already merged). You don't want people to start asking you to add other "future rule changes" like BIP100, BIP101 and BIP102, right?
Also, (and also again) why use Softfork instead of something more generic that also serves for Hardfork deployments?
Maybe "Deployments" or "RuleChanges" would be better.